src/profile/components/ProfilePage.jsx

```diff
@@ -155,98 +161,108 @@ export class ProfilePage extends React.Component {
-                formId="socialLinks"
-                {...commonFormProps}
-              />
+        </div>
+        {this.renderPhotoUploadErrorMessage()}
+        <div className="row">
+          <div className="col-md-4 col-lg-4">
+            <div className="d-none d-md-block mb-4">
+              {this.renderHeadingLockup()}
             </div>
-            <div className="pt-md-3 col-md-8 col-lg-7 offset-lg-1">
-              {shouldShowAgeMessage ? <AgeMessage accountURL="#account" /> : null}
-              <Bio
-                bio={bio}
-                visibilityBio={visibilityBio}
-                formId="bio"
-                {...commonFormProps}
-              />
-              <Certificates
-                visibilityCourseCertificates={visibilityCourseCertificates}
-                formId="certificates"
-                {...commonFormProps}
-              />
+            <div className="d-md-none mb-4">
+              {this.renderViewMyRecordsButton()}
             </div>
+            <Name
+              name={name}
+              visibilityName={visibilityName}
+              formId="name"
+              {...commonFormProps}
+            />
+            <Country
+              country={country}
+              visibilityCountry={visibilityCountry}
+              formId="country"
+              {...commonFormProps}
+            />
+            <PreferredLanguage
+              languageProficiencies={languageProficiencies}
+              visibilityLanguageProficiencies={visibilityLanguageProficiencies}
+              formId="languageProficiencies"
+              {...commonFormProps}
+            />
+            <Education
+              levelOfEducation={levelOfEducation}
+              visibilityLevelOfEducation={visibilityLevelOfEducation}
+              formId="levelOfEducation"
+              {...commonFormProps}
+            />
+            <SocialLinks
+              socialLinks={socialLinks}
+              draftSocialLinksByPlatform={draftSocialLinksByPlatform}
+              visibilitySocialLinks={visibilitySocialLinks}
+              formId="socialLinks"
+              {...commonFormProps}
+            />
+          </div>
+          <div className="pt-md-3 col-md-8 col-lg-7 offset-lg-1">
+            {this.renderAgeMessage()}
+            <Bio
+              bio={bio}
+              visibilityBio={visibilityBio}
+              formId="bio"
+              {...commonFormProps}
+            />
+            <Certificates
+              visibilityCourseCertificates={visibilityCourseCertificates}
+              formId="certificates"
+              {...commonFormProps}
+            />
           </div>
         </div>
       </div>
     );
   }
+
+  render() {
+    return (
+      <div className="profile-page">
+        <Banner />
+        {this.renderContent()}
+      </div>
+    );
+  }
 }
 
 ProfilePage.propTypes = {
+  // Config
+  configuration: PropTypes.shape({
```

Comment:
The profile page, without knowing where the config came from, can identify some URLs that it knows it needs.  This will yell and scream in development if those haven't been properly configured in its parent app, but it doesn't get to know anything about where they actually came from.